Services/LTI/classes/Screen/LtiViewLayoutProvider.php

```diff
@@ -0,0 +1,92 @@
+<?php namespace ILIAS\LTI\Screen;
+
+use ILIAS\GlobalScreen\Scope\Layout\Provider\AbstractModificationProvider;
+use ILIAS\GlobalScreen\Scope\Layout\Provider\ModificationProvider;
+use ILIAS\GlobalScreen\Scope\Layout\Factory\MetaBarModification;
+use ILIAS\GlobalScreen\Scope\Layout\Factory\MainBarModification;
+use ILIAS\GlobalScreen\ScreenContext\Stack\CalledContexts;
+use ILIAS\GlobalScreen\ScreenContext\Stack\ContextCollection;
+use ILIAS\UI\Component\MainControls\MetaBar;
+use ILIAS\UI\Component\MainControls\MainBar;
+use ILIAS\UI\Component\Button\Bulky;
+
+/**
+ * Class LtiViewLayoutProvider
+ *
+ * @author Stefan Schneider <schneider@hrz.uni-marburg.de>
+ */
+class LtiViewLayoutProvider extends AbstractModificationProvider implements ModificationProvider
+{
+
+    /**
+     * @inheritDoc
+     */
+     
+    public function isInterestedInContexts() : ContextCollection
+    {
+        return $this->context_collection->lti();
+    }
+    
+    /**
+     * This is a basic wip implementation for leaving the lti session with an exit button
+     * 
+     * There is an ongoing discussion about a HeaderInfo bar for different ILIAS modes:
+     * 
+     * https://docu.ilias.de/goto_docu_wiki_wpage_5979_1357.html
+     * 
+     * and a PR Feature Discussion:
+     * 
+     * https://github.com/ILIAS-eLearning/ILIAS/pull/2251
+     * 
+     */
+    public function getMetaBarModification(CalledContexts $screen_context_stack) : ?MetaBarModification
+    {
+        $this->dic->logger()->lti()->info("isActive: " . $this->dic["lti"]->isActive());
+        
+        if ($this->dic["lti"]->isActive()) {
```

Comment:
I'm not sure, but is this still needed? As I understand the `ilLTIViewGUI`, the new lti-context is only claimed when LTI is active.

Reply:
Yes it was necessary,  but i will check the conceptual issue again.

Reply:
ok thanks, I check the isInterestedInContext in the meanwhile

Reply:
For testing the LTI Kiosk for Course Object standalone without LTI Consumer:
1. Services/LTI/classes/class.ilLTIViewGUI.php: comment line 117 and uncomment line 120. You can also test some other lti properties with SESSION params like the css injection example.
2. Create a course and a user with login "lti_user" as member. Copy permalink of course 
3. logout and call permalink of course, login with lti_user